config/routes.rb

```diff
@@ -3,7 +3,8 @@
   root to: 'pages#home'
   resources :reviews, only: %i(new create)
   get '/welcome', to: 'pages#welcome'
-  resources :bars
+
+  resources :bars, only:[:index, :show]
```

Comment:
why the only : index + show ? we are going to use all the CRUD elements for the bars